src/team/crazynetwork/raids/Island.java

```diff
@@ -10,17 +10,19 @@
 import java.util.UUID;
 
 public class Island {
-    public static List<Island> islands = new ArrayList<>();
     private int x, y, z; //The coordinates of the island
     private Long raidableTime; //When it will be raidable
     private Player owner; //The owner
     private List<Player> members; //Members of the island
+    private double balence; //Island Balance
```

Comment:
Why is "balance" called "balence"? Makes it harder to understand when reading code from bottom without looking at comments.